websites/B/Blinest/metadata.json

```
@@ -0,0 +1,25 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "828916578698526722",
+		"name": "blinest#3458"
```

Discriminator added by mistake in old `pmd`
```suggestion
		"name": "blinest"
```